docs/manual/src/wasm/configuration.md

```diff
@@ -0,0 +1,58 @@
+# WASM
+
+WASM defines a portable binary format for virtual machines running untrusted code in browsers and various other scenarios. Depending on the scenario, the host language may vary.
+
+As such, `uniffi-rs` does not come with bindings generators "for WASM". This is the job for external uniffi-bindgen generators. e.g. [Typescript][ubrn], [Kotlin][gobley].
+
+[ubrn]: https://github.com/jhugman/uniffi-bindgen-react-native
+[gobley]: https://github.com/gobley/gobley
+
+# Configuration
+
+Configuring the Rust scaffolding ready for WASM binding generators is done by opting into features.
+
+e.g.
+
+```toml
+[dependendencies]
+uniffi = { version = "0.29.2", features = ["wasm-unstable-single-threaded"]}
+```
+
+## Features
+
+### `wasm-unstable-single-threaded`
+
+At time of writing, the state of threading in WASM and Rust support is still in flux.
+
+Much of the ecosystem is built around [gloo][gloo], [`web-sys`][websys] and [`wasm-rayon`][rayon].
+
+Enabling the `wasm-unstable-single-threaded` feature opts out of the `Send` and `Sync` checks when building for `wasm32` target architectures.
+
+It may be safely enabled when building for other architectures.
```

Comment:
Nit: Can you make "safely" a bit more precise?  How about something like "This feature only affects the `wasm32` architecture".